stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/request/QueryChannelPaginationRequest.kt

```diff
@@ -15,6 +15,8 @@ internal class QueryChannelPaginationRequest(var messageLimit: Int = 30) : Query
     var watcherLimit: Int = 30
     var watcherOffset: Int = 0
 
+    var isLoadingMore = false
```

Comment:
I am not sure we really need this property.
The name of this properthy is weird. What mean a pagination query that is not loading more? :thinking:

Reply:
Removed